info.json

```diff
@@ -14,7 +14,6 @@
         }
     ],
     "prerequisite": "",
```

Comment:
why we removed this ? why we can't mark it as false?

Reply:
@cs-suyog-jinturkar By default, For the solution pack that is marked uncertified, this tag was missing in such cases. That's we have removed the certified tag from the info.json

Reply:
@cs-suyog-jinturkar As discussed, I have added a `certified : "false"` entry in info.json